rename_images_by_date.py

```diff
@@ -42,68 +44,123 @@ def findDuplicates(file_dict):
 			
 	return (duplicates, file_dict)
 
-if len(sys.argv) < 2:
-	print("[error] dir name not provided")
-	exit(-1)
+# Create the argument parser, that will display "rename" as program name in output
+my_parser = argparse.ArgumentParser(prog="rename", description='Rename file following given pattern.')
 
+# Add the arguments to parse in the CLI
+#TODO make real alternative option if necessary
+my_parser.add_argument('Dir',
+                       metavar='pathTargetDirectory',
+                       type=str,
+                       help='Idk. Surely target folder to rename pics')
```

Comment:
This is not an option (with double dash "--"). That is the argument required for the running the command line. No need to add "should be mandatory" to the description as it can't left undefined